shared/actions/tracker.js

```diff
@@ -272,10 +288,14 @@ export function onIgnore (username: string): (dispatch: Dispatch, getState: () =
   }
 }
 
+function _getTrackToken (getState, username) {
```

Comment:
it needs the context of username also, or are you saying all the callers need to reach in and get the info